module/metrics/execution.go

```diff
@@ -726,10 +755,24 @@ func (ec *ExecutionCollector) RuntimeTransactionInterpreted(dur time.Duration) {
 	ec.transactionInterpretTime.Observe(float64(dur))
 }
 
-// ChunkDataPackRequested is executed every time a chunk data pack request is arrived at execution node.
-// It increases the request counter by one.
-func (ec *ExecutionCollector) ChunkDataPackRequested() {
-	ec.totalChunkDataPackRequests.Inc()
+// ChunkDataPackRequestProcessed is executed every time a chunk data pack request is picked up for processing at execution node.
+// It increases the request processed counter by one.
+func (ec *ExecutionCollector) ChunkDataPackRequestProcessed() {
+	ec.chunkDataPackRequestProcessedTotal.Inc()
+}
+
+// ChunkDataPackResponseDispatchedInNetwork is executed every time a chunk data pack request is successfully replied.
+// It records the time it took to dispatch the reply on the network.
+// It also increases the response counter by one.
+func (ec *ExecutionCollector) ChunkDataPackResponseDispatchedInNetwork(dispatchTime time.Duration) {
+	ec.chunkDataPackResponseDispatchTime.Observe(float64(dispatchTime.Milliseconds()))
+	ec.chunkDataPackResponseDeliveredTotal.Inc()
+}
+
+// ChunkDataPackRetrievedFromDatabase is executed every time a chunk data pack is queried from database.
+// It records the time it took to query the chunk data pack.
+func (ec *ExecutionCollector) ChunkDataPackRetrievedFromDatabase(queryTime time.Duration) {
+	ec.chunkDataPackQueryTime.Observe(float64(queryTime.Milliseconds()))
```

Comment:
```suggestion
	ec.chunkDataPackQueryTime.Observe(float64(queryTime.Seconds()))
```

Consistency with prometheus naming